driver/fixtures/fixtures.go

```diff
@@ -410,6 +423,9 @@ func isBench(name, ext string) (string, bool) {
 
 func (s *Suite) benchmarkFixtures(b *testing.B) {
 	b.StopTimer()
+	if !runsInDocker() {
```

Comment:
Maybe do this first? (Not that it really matters when you stop the timer, but there's no point in doing anything for a benchmark that isn't going to run)